### DISCUSSION TITLE
Rider plugin run configuration with matching program arguments

### DISCUSSION THREAD

**Discussion by jwfx** [nuke-build/nuke#1034] [Q&A] [answered]
When selecting `Debug without dependencies` on a target, a run configuration is created. It would be very helpful if the `without dependencies` option is selected, that the run configuration would actually be created with `--skip --target <target>` as program arguments. This way when you are done debugging, you could just re-run the same run configuration and have the same behavior (running the target without dependencies) again.

With the current plugin version 2022.2.0, is there a way to achieve this behavior?

![image](https://user-images.githubusercontent.com/39484381/195342489-4d9484a0-9ddf-423e-a178-eaf86c0bf64d.png)

![image](https://user-images.githubusercontent.com/39484381/195342740-53a97afc-b8db-4f8d-a2e5-3d89b2d12e42.png)

**Comment by matkoch** [comment, marked as answer]
no, and also not planned

**Reply by jwfx** [reply]
Thanks for your fast response.

Would it be unreasonable to request this small quality of life improvement? Unless of course there is another way the plugin should be used that I'm currently missing.

Also, as far as I can tell the itself plugin is not open source, correct?

**Reply by matkoch** [reply]
That's correct. If you're interested in something being developed, then there are sponsorship tiers listed here: https://github.com/sponsors/matkoch
